planning/autoware_manual_lane_change_handler/CMakeLists.txt

```
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.14)
+project(autoware_manual_lane_change_handler)
+
+find_package(ament_cmake_auto REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(pluginlib REQUIRED)
+find_package(autoware_planning_msgs REQUIRED)
+
+ament_auto_find_build_dependencies()
```

For autoware packages, we recommend using [autoware_cmake](https://github.com/autowarefoundation/autoware_cmake/tree/main/autoware_cmake)

You would have to update package.xml to have  `<buildtool_depend>autoware_cmake</buildtool_depend>` as dependency as well.

```suggestion
find_package(autoware_cmake REQUIRED)
autoware_package()
```